sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientMockTests.cs

```diff
@@ -229,8 +229,152 @@ public async Task AnalyzeDocumentFromUriEncodesBlankSpaces()
             }
         }
 
+        [Test]
+        public async Task AnalyzeDocumentCanParseDocumentFieldWithBooleanValue()
```

Comment:
We still haven't gotten proper files to test these features, so we're adding mock tests for them.